public/app/AppWrapper.tsx

```diff
@@ -107,6 +107,7 @@ export class AppWrapper extends Component<AppWrapperProps, AppWrapperState> {
                   <CompatRouter>
                     <ModalsContextProvider>
                       <GlobalStyles />
+                      <PortalContainer />
```

Comment:
This seems.... fishy to me. I would not expect this to fix the issue.

I'm guessing there's some broken styles in the role picker....

Reply:
It seems like the `grafana-app` container pushes the portal container all the way to the button of the page, where previously it's been at the top. Moving the portal to the top fixes the issue, but I'm not sure of it breaks anything else.